resources/[managers]/spawnmanager/spawnmanager.lua

```diff
@@ -306,7 +312,9 @@ function spawnPlayer(spawnIdx, cb)
 
         ShutdownLoadingScreen()
 
-        if IsScreenFadedOut() then
+        -- if skipFade is specified, we should not be meddling in fading at all
+        -- NOTE: Could this be a breaking change for some?
+        if IsScreenFadedOut() and not spawn.skipFade then
```

Comment:
Please make this a new parameter like `skipFadeIn`.

Reply:
Called it exactly that.